packages/web-client/package.json

```diff
@@ -94,6 +94,7 @@
     "ember-cli-deploy-display-revisions": "^2.1.2",
     "ember-cli-deploy-revision-data": "^1.0.0",
     "ember-cli-deploy-s3": "^3.0.0",
+    "ember-cli-deploy-s3-index": "^1.3.1",
```

Comment:
How is the s3-index plugin different from the s3 plugin?

Reply:
In the “lightning” deployment strategy, the `index.html` is written separately from the assets. So I was basing this on that despite us not needing it to be deployed separately. BUT I now see in the `ember-cli-deploy-s3` documentation:

> It could be used to upload the assets (js, css, images etc) or indeed the application's index.html.

So I’ll try that now, thanks.